build.sbt

```diff
@@ -586,6 +594,63 @@ lazy val testJunitRunnerTests = project.module
         .value
   )
 
+lazy val testJunitEngine = project.module
+  .in(file("test-junit-engine"))
+  .settings(stdSettings("zio-test-junit-engine"))
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.junit.jupiter"       % "junit-jupiter-api"       % "5.11.0",
```

Comment:
Is this dependency necessary? I couldn't figure out where it's being used

Reply:
Indeed